request-filterer/request-filterer-impl/src/main/java/com/duckduckgo/request/filterer/impl/RequestFiltererImpl.kt

```diff
@@ -81,6 +81,7 @@ class RequestFiltererImpl @Inject constructor(
         currentPage = url
         hasTimeElapsed = false
         if (job?.isActive == true) job?.cancel()
+        // scope is created with context dispatchers.io()
```

Comment:
Rather than adding a comment, we could be explicit in using `dispatchers.io()` here. (no strong feelings)

Reply:
Not sure I understand/agree with this logic. `scope` is a private property in an impl class. I don't think we need to start guarding mis-usages of private properties/funs inside the same class. Seems unnecessary.

Reply:
It's more that you felt the need to add a comment at all, which indicates the line of code itself isn't self-explanatory. To make the line of code clearer in its intent, you can either comment it (like you have) or explicitly add the dispatcher which is more self-documenting.

Doesn't matter too though; the comment also makes the intent of the line clear.